treeherder/model/models.py

```diff
@@ -467,7 +468,7 @@ class Job(models.Model):
     result = models.CharField(max_length=25)
     state = models.CharField(max_length=25)
 
-    submit_time = models.DateTimeField()
+    submit_time = models.DateTimeField(db_index=True)
```

Comment:
So sorry for the churn on this, but it occurred to me that perhaps what legacy OrangeFactor does is wrong, and we should maybe copy what Perfherder does. I've asked Geoff/Joel/Will for feedback here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1367362#c14